t/t-track.sh

```diff
@@ -711,3 +711,26 @@ begin_test "track: escaped glob pattern with spaces in .gitattributes"
   assert_pointer "main" "$filename" "$contents_oid" 15
 )
 end_test
+
+begin_test "--json output"
+(
+  set -e
+
+  reponame="track-json"
+  git init "$reponame"
+  cd "$reponame"
+
+  git lfs track '*.dat'
+  git lfs track --lockable '*.bin'
+  echo 'a.dat !filter' >>.gitattributes
+  git add .gitattributes
```

Comment:
Out of interest, is this necessary?  I tried a `git lfs track` on an untracked `.gitattributes` and it worked, so it may not need to be added-but-not-committed here.

Reply:
Technically, no.  I can remove it.